### PR TITLE
Updates to MyAccount API guide

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description
- **What's changed?** Changes to the guide to reflect changes to the API reference doc
- **Is this PR related to a Monolith release?** No

### Resolves

* [OKTA-521952](https://oktainc.atlassian.net/browse/OKTA-521952)

### CODE REVIEW

**Review by hanyuanzhang-okta — APPROVED** [review]
LGTM

**Review by JayHe-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/configure-user-scoped-account-management/main/index.md [hunk 1]
```diff
@@ -56,6 +56,14 @@ A valid API version in the `Accept` header is required to access the API. Curren
 Accept: application/json; okta-version=1.0.0
 ```
 
+## Access Token assurance
+
+MyAccount operations that create, update, or delete resources require Access Tokens that are 15 minutes old or younger. API calls with Access Tokens older than 15 minutes require re-authentication. If you don't re-authenticate the token, the API returns a 403 error with the following content in the header:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-user-scoped-account-management/main/index.md:61]:
```suggestion
MyAccount operations that create, update, or delete resources require access tokens that are 15 minutes old or younger. API calls with access tokens older than 15 minutes require re-authentication. If you don't re-authenticate the token, the API returns a 403 error with the following content in the header:
```

**Review by susanharper-okta — APPROVED** [review]
one comment - approving to not hold you up

File: packages/@okta/vuepress-site/docs/guides/configure-user-scoped-account-management/main/index.md [hunk 2]
```diff
@@ -56,6 +56,14 @@ A valid API version in the `Accept` header is required to access the API. Curren
 Accept: application/json; okta-version=1.0.0
 ```
 
+## Access Token assurance
+
+MyAccount operations that create, update, or delete resources require access tokens that are 15 minutes old or younger. API calls with access tokens older than 15 minutes require re-authentication. If you don't re-authenticate the token, the API returns a 403 error with the following content in the header:
+
+```json
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-user-scoped-account-management/main/index.md:63]:
```suggestion
```bash
```

**Pull request merged**